apps/site/util/partners/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as PartnersLogo from '@node-core/ui-components/Icons/PartnerLogos';
+import type { ElementType } from 'react';
+
+import type { Partners } from '#site/types';
+
+/**
+ * Creates an icon element for a component
+ */
+const createIcon = (
+  IconModule: Record<string, Record<string, ElementType>>,
+  iconName: string,
+  type?: 'Logo' | 'Favicon'
```

Comment:
```suggestion
  type: 'Logo' | 'Favicon' = 'Favicon'
```

Better than `[type || 'Favicon'];`